stories/components/footer/footer.stories.js

```diff
@@ -7,9 +7,15 @@ import "../social-nav/social-nav.scss";
 import "../button/button.scss";
 import "../section/section.scss";
 
+import SocialNavContent from "../social-nav/social-nav.content.json";
```

Comment:
@mejiaj since the footer is used on all the pages, don't we need to include this in all their stories.js too?

Reply:
Yes, great catch. Fixed in d2e140b.